README.md

```diff
@@ -109,6 +113,19 @@ YOLayout is a great fit for custom drawn controls. Views that render in `drawRec
 
 Nope. If your layout is really simple, or it doesn't have dynamic sizing, just use `layoutSubviews` (UIKit) or `layout` (AppKit) like you normally would. YOLayout doesn't do anything special to override behavior and is compatible with existing layout methods.
 
+## Disadvantages of using YOLayout
+
+YOLayout, like most things, has trade-offs. We like using it especially for really complex layouts with lots of different alignments and for things that are dynamically sized. 
+
+But there are downsides:
+
+- You might find yourself hardcoding pixel values for things like padding, a lot.
+- If you are tweaking padding and positioning, you have to re-run the project to visualize those changes, whereas in Interface Builder you would see those changes immediately.
```

Comment:
Not necessarily if you make your view an `IB_DESIGNABLE` and all your subviews are drawable! I've been doing this a lot lately so I can test views without a full compile. You can make a throwaway xib and instantiate your view there to get a preview rendering.

That said, that's pretty nuanced, so we should probably still keep this as a disadvantage.